vmware-base/src/main/java/com/cloud/hypervisor/vmware/mo/HypervisorHostHelper.java

```diff
@@ -153,6 +153,48 @@
     public static final String VSPHERE_DATASTORE_BASE_FOLDER = "fcd";
     public static final String VSPHERE_DATASTORE_HIDDEN_FOLDER = ".hidden";
 
+    protected final static Map<String, Integer> apiVersionHardwareVersionMap;
+
+    static {
+        apiVersionHardwareVersionMap = new HashMap<String, Integer>();
```

Comment:
Is this mapping used only for comparing hosts versions? Maybe we can remove it and do a string comparission?

Reply:
@nvazquez not just comparison, minimum hardware version is needed when migration is between different hosts so the worker VM can be created with lower hardware version